gonna look why the hashes mismatch soon

## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 512845`
Commit: [`578129075b650fa3eb2fba796b370f96631bf354`](https://github.com/NixOS/nixpkgs/commit/578129075b650fa3eb2fba796b370f96631bf354) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/578129075b650fa3eb2fba796b370f96631bf354..pull/512845/head))
Merge: [`298fa2c685b194be337a7b0275cb5856df32bb1e`](https://github.com/NixOS/nixpkgs/commit/298fa2c685b194be337a7b0275cb5856df32bb1e)

Logs: https://github.com/kybe236/nixpkgs-review-gha/actions/runs/24849413987


---
### `x86_64-linux`
<details>
  <summary>:x: 31 packages failed to build:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>bash-preexec</li>
    <li>bats</li>
    <li>crypt4gh (python313Packages.crypt4gh)</li>
    <li>crypt4gh.dist (python313Packages.crypt4gh.dist)</li>
    <li>devmode</li>
    <li>image_optim</li>
    <li>jpeg-archive</li>
    <li>kicad</li>
    <li>kicad-testing</li>
    <li>kicad-unstable</li>
    <li>kicadAddons.kikit</li>
    <li>kicadAddons.kikit-library</li>
    <li>kikit</li>
    <li>kikit.dist</li>
    <li>last</li>
    <li>locate-dominating-file</li>
    <li>packcc</li>
    <li>parallel</li>
    <li>parallel-full</li>
    <li>parallel-full.doc</li>
    <li>parallel-full.man</li>
    <li>parallel.doc</li>
    <li>parallel.man</li>
    <li>plata-theme</li>
    <li>profile-cleaner</li>
    <li>python314Packages.crypt4gh</li>
    <li>python314Packages.crypt4gh.dist</li>
    <li>scap-security-guide</li>
    <li>serenityos-emoji-font</li>
    <li>wikiman</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 30 packages failed to build:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>bash-preexec</li>
    <li>bats</li>
    <li>crypt4gh (python313Packages.crypt4gh)</li>
    <li>crypt4gh.dist (python313Packages.crypt4gh.dist)</li>
    <li>devmode</li>
    <li>image_optim</li>
    <li>jpeg-archive</li>
    <li>kicad</li>
    <li>kicad-testing</li>
    <li>kicad-unstable</li>
    <li>kicadAddons.kikit</li>
    <li>kicadAddons.kikit-library</li>
    <li>kikit</li>
    <li>kikit.dist</li>
    <li>locate-dominating-file</li>
    <li>packcc</li>
    <li>parallel</li>
    <li>parallel-full</li>
    <li>parallel-full.doc</li>
    <li>parallel-full.man</li>
    <li>parallel.doc</li>
    <li>parallel.man</li>
    <li>plata-theme</li>
    <li>profile-cleaner</li>
    <li>python314Packages.crypt4gh</li>
    <li>python314Packages.crypt4gh.dist</li>
    <li>scap-security-guide</li>
    <li>serenityos-emoji-font</li>
    <li>wikiman</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>bash-preexec</li>
    <li>bats</li>
    <li>crypt4gh (python313Packages.crypt4gh)</li>
    <li>crypt4gh.dist (python313Packages.crypt4gh.dist)</li>
    <li>devmode</li>
    <li>image_optim</li>
    <li>jpeg-archive</li>
    <li>last</li>
    <li>locate-dominating-file</li>
    <li>packcc</li>
    <li>parallel</li>
    <li>parallel-full</li>
    <li>parallel-full.doc</li>
    <li>parallel-full.man</li>
    <li>parallel.doc</li>
    <li>parallel.man</li>
    <li>profile-cleaner</li>
    <li>python314Packages.crypt4gh</li>
    <li>python314Packages.crypt4gh.dist</li>
    <li>wikiman</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>packcc</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>bash-preexec</li>
    <li>bats</li>
    <li>crypt4gh (python313Packages.crypt4gh)</li>
    <li>crypt4gh.dist (python313Packages.crypt4gh.dist)</li>
    <li>devmode</li>
    <li>image_optim</li>
    <li>jpeg-archive</li>
    <li>locate-dominating-file</li>
    <li>parallel</li>
    <li>parallel-full</li>
    <li>parallel-full.doc</li>
    <li>parallel-full.man</li>
    <li>parallel.doc</li>
    <li>parallel.man</li>
    <li>profile-cleaner</li>
    <li>python314Packages.crypt4gh</li>
    <li>python314Packages.crypt4gh.dist</li>
    <li>serenityos-emoji-font</li>
    <li>wikiman</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>packcc</summary>
<pre>      (from function `run&#x27; in file /nix/store/zrswvpb9j5pb1fwq79cy2kxwv1h7xz5f-bats-1.12.0/lib/bats-core/test_functions.bash, line 418,
       from function `run_for_input&#x27; in file utils.sh, line 16,
       in test file strings.d/test.bats, line 13)
BW01: `run`&#x27;s command `timeout 5s /nix/build/nix-16722-1197572122/source/tests/strings.d/parser` exited with code 127, indicating &#x27;Command not found&#x27;. Use run&#x27;s return code checks, e.g. `run -127`, to fix this message.
      (from function `run&#x27; in file /nix/store/zrswvpb9j5pb1fwq79cy2kxwv1h7xz5f-bats-1.12.0/lib/bats-core/test_functions.bash, line 418,
       from function `run_for_input&#x27; in file utils.sh, line 16,
       in test file strings.d/test.bats, line 16)
BW01: `run`&#x27;s command `timeout 5s /nix/build/nix-16722-1197572122/source/tests/strings.d/parser` exited with code 127, indicating &#x27;Command not found&#x27;. Use run&#x27;s return code checks, e.g. `run -127`, to fix this message.
      (from function `run&#x27; in file /nix/store/zrswvpb9j5pb1fwq79cy2kxwv1h7xz5f-bats-1.12.0/lib/bats-core/test_functions.bash, line 418,
       from function `run_for_input&#x27; in file utils.sh, line 16,
       in test file strings.d/test.bats, line 19)
BW01: `run`&#x27;s command `timeout 5s /nix/build/nix-16722-1197572122/source/tests/util_eol.d/parser` exited with code 127, indicating &#x27;Command not found&#x27;. Use run&#x27;s return code checks, e.g. `run -127`, to fix this message.
      (from function `run&#x27; in file /nix/store/zrswvpb9j5pb1fwq79cy2kxwv1h7xz5f-bats-1.12.0/lib/bats-core/test_functions.bash, line 418,
       from function `run_for_input&#x27; in file utils.sh, line 16,
       in test file util_eol.d/test.bats, line 13)
BW01: `run`&#x27;s command `timeout 5s /nix/build/nix-16722-1197572122/source/tests/util_tab.d/parser` exited with code 127, indicating &#x27;Command not found&#x27;. Use run&#x27;s return code checks, e.g. `run -127`, to fix this message.
      (from function `run&#x27; in file /nix/store/zrswvpb9j5pb1fwq79cy2kxwv1h7xz5f-bats-1.12.0/lib/bats-core/test_functions.bash, line 418,
       from function `run_for_input&#x27; in file utils.sh, line 16,
       in test file util_tab.d/test.bats, line 13)
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

--- a/pkgs/by-name/pa/parallel/package.nix
+++ b/pkgs/by-name/pa/parallel/package.nix
@@ -12,11 +12,11 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "parallel";
-  version = "20260322";
+  version = "20260422";
 
   src = fetchurl {
     url = "mirror://gnu/parallel/parallel-${finalAttrs.version}.tar.bz2";
-    hash = "sha256-dkaA6TL00NIc8DKb2fnu1lmJXeFoNgAfZJFTO4Ir7+A=";
+    hash = "sha256-ZkzxZdZuohey9JzZanhl7PkMnQYWWZzCq6jK1IHZB7s=";
   };
 
   outputs = [
@@ -56,6 +56,13 @@ stdenv.mkDerivation (finalAttrs: {
       }"
   '';
 
+  # Force run `check` instead of the `tests` target, because the `test` target depends on a private testsuite.
+  # The reason `check` isn't used by default is due to it failing when selecting the target to run. (due to stdenv not overriding the Makefiles SHELL variable from `/bin/bash`).
+  #
+  # See:
+  #   https://github.com/NixOS/nixpkgs/blob/ef4f672aa2be8b268a4280e8e2a68cd97a4cf67b/pkgs/stdenv/generic/setup.sh#L1541
+  #   https://github.com/NixOS/nixpkgs/blob/ef4f672aa2be8b268a4280e8e2a68cd97a4cf67b/pkgs/stdenv/generic/setup.sh#L1555
+  checkTarget = "check";
   doCheck = true;
 
   meta = {

